http/src/routing/route.rs

```diff
@@ -387,6 +387,11 @@ pub enum Route<'a> {
     GetCurrentUserApplicationInfo,
     /// Route information to get the current user.
     GetCurrentUser,
+    /// Route information to get the current user as a member object within a guild.
+    GetCurrentUserGuildMember {
+        /// The ID of the guild.
```

Comment:
```suggestion
        /// ID of the guild.
```

Reply:
Usually, I would just commit this change and merge, but seems that the owner of your fork and your user account being different doesn't allow this.